fastos/src/vespa/fastos/timestamp.h

```diff
@@ -59,25 +59,27 @@ class ClockSystem
 {
 public:
     static int64_t now();
-    static int64_t adjustTick2Sec(int64_t tick) { return tick; }
 };
 
-template <typename ClockT>
-class StopWatchT : public ClockT
+class ClockSteady
 {
 public:
-    StopWatchT(void) : _startTime(), _stopTime() { }
+    static TimeStamp now();
```

Comment:
why do the different clocks have different return types?

Reply:
A step on the way, will get there.